inc/sccmdb.class.php

```diff
@@ -57,6 +57,11 @@ public function connect()
             "CharacterSet" => "UTF-8",
         ];
 
+        // Check SSL certificate verification setting
+        if ($PluginSccmConfig->getField('verify_ssl_cert') != "1") {
+            $connectionOptions["TrustServerCertificate"] = true;
+        }
```

Comment:
```suggestion
        if ($PluginSccmConfig->getField('verify_ssl_cert') != "1") {
            $connectionOptions["TrustServerCertificate"] = false;
        }
```

verify_ssl_cert = 0 = do not verify